docs/src/components/ComponentVariableTable.tsx

```diff
@@ -51,7 +52,9 @@ export const ComponentVariableTable = ({ componentName }) => {
 
   return (
     <View className="docs-css-variables">
-      <Table variation="bordered">{variableNames}</Table>
+      <Table variation="bordered">
```

Comment:
Probably not a great use for `table` here  (since it's just a list of names, not really tabular data) but fixed it to clear the warning. We should probably reconsider how we can present this list semantically in a bordered responsive grid view.

Reply:
Maybe `Collection` primitive? By default it is a list `Collection`